build.cake

```diff
@@ -825,14 +853,26 @@ Task("Test")
     .IsDependentOn("TestEngine")
     .IsDependentOn("TestConsole");
 
+Task("BuildPackages")
+    .Description("Builds all packages for distribution")
+    .IsDependentOn("BuildNuGetPackages")
+    .IsDependentOn("BuildChocolateyPackages")
+    .IsDependentOn("BuildMsiPackage")
+    .IsDependentOn("BuildZipPackage");
+
+Task("TestPackages")
+    .Description("Tests the packages")
+    .IsDependentOn("TestNugetPackages")
```

Comment:
Niggle I've just spotted - can we case the `BuildNuGetPackages` and `TestNugetPackages` targets consistently? I'm not too fussy which way round - `NuGet` would be consistent with the branding and Cake Methods etc.

Reply:
Fixed!